forge-gui/res/cardsfolder/upcoming/oskar_rubbish_reclaimer.txt

```diff
@@ -0,0 +1,10 @@
+Name:Oskar, Rubbish Reclaimer
+ManaCost:3 U B
+Types:Legendary Creature Human Wizard
+PT:3/3
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each card type among cards in your graveyard.

```

Comment:
Description still wrong...

Reply:
Yeah, I had both wrong and missed the first one. Oops!